dash_bio/__init__.py

```diff
@@ -115,7 +110,5 @@
 _css_dist = []
```

Comment:
This line can be dropped as well now... not a big deal though :)

Reply:
Thanks for the catch! 

:hocho:  in 02144f9